include/reactor-cpp/scheduler.hh

```diff
@@ -27,123 +28,117 @@ namespace reactor {
 class Scheduler;
 class Worker;
 
-class Worker {
- public:
-  Scheduler& scheduler;
-  const unsigned id{0};
-  std::thread thread;
+class Worker { // NOLINT
+public:
+  Scheduler& scheduler_;
```

Comment:
In the *.cc files, the `&` and `*` would be placed next to the identifier and not next to the type. Why is this different in the header files?